### PR TITLE
feat: support action name attribute

### PR DESCRIPTION
**Pull request by larmitage-bjss**
To be merged after #525 

Added an option to include the name of the action as a header, defaulting to not show

This is so that when we add functionality for documenting a workflow there is a header at the top as a workflow doesn't have a description

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -40,6 +40,11 @@ const args = await yargs(process.argv.slice(2))
       type: "string",
       alias: "l",
     },
+    "include-name-header": {
+      description: "Include a header with the action/workflow name",
+      type: "boolean",
+      alias: "h",
```
Comment (npalm) [inline src/cli.ts:46]: Maybe better to avoid the `h` since it is a common alias for help. We could use -n from name or switch to 2 letters.

**Review by npalm — CHANGES_REQUESTED** [review]
@larmitage-bjss Thx! The PR looks good, took the freedom to rebase the PR. Have made one request for change.

File: src/cli.ts (hunk 1 above)
Comment (larmitage-bjss) [reply src/cli.ts:46]: Makes sense, thanks I'll update

**Comment by larmitage-bjss** [comment]
If it's two characters the shorthand needs two dashes, so I've chosen just `-n`. Ready for re-review please

**Review by npalm — APPROVED** [review]
Thx for you great contribution

**Pull request merged**